core/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala

```diff
@@ -751,6 +751,27 @@ class ApplicationInfoSuite extends FunSuite with Logging {
     assert(execInfo.head.maxMem === 5538054144L)
   }
 
+  test("test malformed json eventlog") {
```

Comment:
Suggest a different name to reflect the real purpose of that unit-test. the current string may cause someone to think that the unit-test is just testing against malformated eventlogs.
Also, add a comment explaining the unit-test logic and what it tests.

Reply:
Thanks @amahussein! Updated this.